Security/Guidelines/OpenSSH.mediawiki

```diff
@@ -364,7 +380,33 @@ These are not officially supported and may require you to recompile OpenSSH.
 * [https://github.com/devinteske/apple/tree/master/OpenSSH-186/openssh OpenSSH MacOS X] (see also http://devinteske.com/)
 }}
 
-=== Safer alternative to SSH agent forwarding ===
+=== Recommended, safer alternatives to SSH agent forwarding ===
+
+==== OpenSSH >=7.3  ====
+
+OpenSSH 7.3 onwards allow users to jump through several hosts in a rather automated fashion. It has fully support for
+scp and sftp commands as well as regular ssh.
+
+For example to reach an host being a bastion/jumphost:
+<source code="bash">
+# Single jump
+$ ssh -J ssh.mozilla.com myhost.private.scl3.mozilla.com
+
+# Jump through 2 hops
```

Comment:
This comment in markdown is being interpreted as a numbered list and I don't think that is what you are going for here.